ext/mbstring/mbstring.c

```diff
@@ -3217,6 +3209,14 @@ PHP_FUNCTION(mb_levenshtein)
 	unsigned char *in_2 = (unsigned char*)ZSTR_VAL(string2);
 	unsigned int state = 0;
 
+	if (strlen_1 == 0) {
+		RETVAL_LONG(strlen_2 * cost_ins);
```

Comment:
Note that these should be RETURN_LONG too because otherwise the code will keep executing.

Reply:
Thanks again. fixed.